lib/verifier/post_zephir_verifier.rb

```diff
@@ -0,0 +1,195 @@
+    #  empty
+    def verify_dollar_dup(date: current_date)
+      dollar_dup = self.class.dated_derivative(location: :TMPDIR, name: "vufind_incremental_YYYY-MM-DD_dollar_dup.txt.gz", date: date)
+      if verify_file(path: dollar_dup)
+        gz_count = gzip_linecount(path: dollar_dup)
+        if gz_count.positive?
+          error message: "spurious dollar_dup lines: #{dollar_dup} should be empty (found #{gz_count} lines)"
+        end
+      end
+    end
+
+    # Frequency: MONTHLY
+    # Files:
+    #   INGEST_BIBRECORDS/groove_full.tsv.gz
+    #   INGEST_BIBRECORDS/zephir_ingested_items.txt.gz
+    # Contents: TODO
+    # Verify: readable
+    def verify_ingest_bibrecords(date: current_date)
+      if date.last_of_month?
+        verify_file(path: self.class.derivative(location: :INGEST_BIBRECORDS, name: "groove_full.tsv.gz"))
+        verify_file(path: self.class.derivative(location: :INGEST_BIBRECORDS, name: "zephir_ingested_items.txt.gz"))
+      end
+    end
+
+    # Frequency: BOTH
+    # Files:
+    #   RIGHTS_ARCHIVE/zephir_upd_YYYYMMDD.rights (daily)
+    #   RIGHTS_ARCHIVE/zephir_full_YYYYMMDD.rights (monthly)
+    # Contents: see verify_rights_file_format
+    # Verify:
+    #   readable
+    #   accepted by verify_rights_file_format
+    def verify_rights(date: current_date)
+      upd_path = self.class.dated_derivative(location: :RIGHTS_ARCHIVE, name: "zephir_upd_YYYYMMDD.rights", date: date)
+      if verify_file(path: upd_path)
+        verify_rights_file_format(path: upd_path)
+      end
+
+      if date.last_of_month?
+        full_path = self.class.dated_derivative(location: :RIGHTS_ARCHIVE, name: "zephir_full_YYYYMMDD.rights", date: date)
+        if verify_file(path: full_path)
+          verify_rights_file_format(path: full_path)
+        end
+      end
+    end
+
+    # Rights file must:
+    # * exist & be be readable (both covered by verify_rights)
+    # * either be empty, or all its lines must match regex.
+    def verify_rights_file_format(path:)
+      regex = /^ [a-z0-9]+ \. [a-z0-9:\/\$\.]+ # col 1, namespace.objid
+              \t (ic|pd|pdus|und)              # col 2, one of these
+              \t bib                           # col 3, exactly this
+              \t bibrights                     # col 4, exactly this
+              \t [a-z]+(-[a-z]+)*              # col 5, digitization source, e.g. 'ia', 'cornell-ms'
+              $/x
+
+      # This allows an empty file as well, which is possible.
+      File.open(path) do |f|
+        f.each_line do |line|
+          line.strip!
+          unless line.match?(regex)
+            error message: "Rights file #{path} contains malformed line: #{line}"
+          end
+        end
+      end
+    end
+
+    # Frequency: MONTHLY
+    # Files:
+    #   ZEPHIR_DATA/full/zephir_full_monthly_rpt.txt
+    #   ZEPHIR_DATA/full/zephir_full_YYYYMMDD.rights_rpt.tsv
+    # Contents: TODO
+    # Verify: readable
+    def verify_zephir_data(date: current_date)
```

Comment:
I'd be inclined to remove this since we aren't using those files the way they are (maybe exporting what's in there as prometheus metrics or something in the future would be useful)